JitsiMeetJS.js

```diff
@@ -147,6 +147,31 @@ var LibJitsiMeet = {
     isDeviceChangeAvailable: function () {
         return RTC.isDeviceChangeAvailable();
     },
+    /**
+     * Returns true if changing the audio output of media elements is supported
+     * and false if not.
+     */
+    isAudioOutputDeviceChangeAvailable: function () {
```

Comment:
Yep, I will remove 'isAudioOutputDeviceChangeAvailable' from the JitsiMeetJS object.
Instead will make 'isDeviceChangeAvailable' to accept optional argument 'deviceType'. If value of this argument will be 'input' or undefined, it will work as before, in case of 'output' value - will use internal RTC.isAudioOutputDeviceChangeAvailable method.